docs/source/conceptual_guides/lora.mdx

```diff
@@ -49,6 +49,8 @@ As with other methods supported by PEFT, to fine-tune a model using LoRA, you ne
 - `alpha`: LoRA scaling factor.
 - `bias`: Specifies if the `bias` parameters should be trained. Can be `'none'`, `'all'` or `'lora_only'`.
 - `modules_to_save`: List of modules apart from LoRA layers to be set as trainable and saved in the final checkpoint. These typically include model's custom head that is randomly initialized for the fine-tuning task.
+- `layers_to_transform`: List of layers to be transformed by LoRA. If not specified, all layers in `target_modules` are transformed.
```

Comment:
Nice, this is super clean! ✨ 

Should we also add these two parameters to the docstring [here](https://github.com/huggingface/peft/blob/4fd374e80d670781c0d82c96ce94d1215ff23306/src/peft/tuners/lora.py#L48)?

Reply:
Awesome! Yes will add it now

Reply:
Added it in [eb16fc3](https://github.com/huggingface/peft/pull/429/commits/eb16fc35a1ede25bdeb2501740ed3a0c2f9166ee) !

Reply:
Very nice, thank you!